### DISCUSSION TITLE
Invalid parameter not satisfying: point.x != INFINITY && point.y != INFINITY

### DISCUSSION THREAD

**Discussion by billychen2014** [appium/appium#18967] [Q&A]
hello,

i'm always get such error while executing command below:

```
driver.execute_script("mobile: dragFromToForDuration",
                                {"duration": 1.5, "fromX": 33, "fromY": 278, "toX": 144, "toY": 278}
```
details server log can be found here:
https://gist.github.com/billychen2014/078b6ed9318ff0fc24a3def4bec55378

any suggestions ? 

server: appium 2.0.1
driver: xcuitest@4.33.1
platform: iOS

**Comment by KazuCocoa** [comment]
What was the device screen size?

**Reply by billychen2014** [reply]
width: 375 height:812 and scale is 3.0(pixelRaito)
device :iPhone mini12

**Comment by billychen2014** [comment]
it looks like this problem can be resolved after upgrading 'Appium-Python-Client ' to 2.11.1 and 'selenium' to 4.11.2.
